frontend/src/components/ApplicationSummaryTable.js

```diff
@@ -10,37 +12,52 @@ import Paper from '@mui/material/Paper';
 function createData(name, answer) {
   return { name, answer };
 }
-const rows = [
-  createData('Application ID:', 1234567),
-  createData('Last name / surname:', 'Smith'),
-  createData('First name / given name:', 'John'),
-  createData('Middle name(s):', 'David'),
-  createData('Email address:', 'jdsmith@shaw.ca'),
-  createData('Date of birth:', '19700906'),
-  createData('Street address:', '910 Government St.'),
-  createData('City:', 'Victoria BC'),
-  createData('Postal Code:', 'V8W 3Y8'),
-  createData('Social Insurance Number (SIN):', '******123'),
-  createData("B.C. Driver's Licence number:", 'DL:2222222'),
-  createData('Tax Year:', '2020'),
-  createData(
-    'Consent to Disclosure and Storage of, and Access to, Personal Information:',
-    'BCEIDJSMITH 20220504 12:04:01 PST'
-  ),
-  createData(
-    'Consent to Disclosure of Information from Income Tax Records:',
-    'BCEIDJSMITH 20220504 12:04:01 PST'
-  )
-];
 
 const ApplicationFormDetails = (props) => {
-  console.log(props);
+  const { id } = props;
+  const axiosInstance = useAxios();
+  const enabled = true;
+  const queryFn = () =>
+    axiosInstance.current.get(`/api/application-form/${id}`);
+
+  const { data } = useQuery({
+    id,
+    queryFn,
+    enabled
+  });
+
+  const rows = [
+    createData('Application ID:', data ? data.data.id : ''),
+    createData('Last name / surname:', data ? data.data.last_name : ''),
+    createData('First name / given name:', data ? data.data.first_name : ''),
+    createData('Middle name(s):', data ? data.data.middle_names : ''),
+    createData('Email address:', data ? data.data.email : ''),
+    createData('Date of birth:', data ? data.data.date_of_birth : ''),
+    createData('Street address:', data ? data.data.address : ''),
+    createData('City:', data ? data.data.city : ''),
+    createData('Postal Code:', data ? data.data.postal_code : ''),
+    createData('Social Insurance Number (SIN):', data ? data.data.sin : ''),
+    createData(
+      "B.C. Driver's Licence number:",
+      data ? data.data.drivers_licence : ''
+    ),
+    createData('Tax Year:', data ? data.data.tax_year : ''),
+    createData(
+      'Consent to Disclosure and Storage of, and Access to, Personal Information:',
+      data ? data.data.consentpersonal : ''
+    ),
+    createData(
+      'Consent to Disclosure of Information from Income Tax Records:',
+      data ? data.data.consenttax : ''
+    )
+  ];
+
   return (
     <Box>
       <h3>Individual Application Confirmation</h3>
       <p>
         Print this page for your records. You will also receive an email
-        confirmation at jdsmith@shaw.ca
+        confirmation at {data && data.data && data.data.email}
```

Comment:
if you've checked isloading/iserror previously you won't have to do all these && checks, the data will be there. I'd probably destructure the above as well to avoid data.data